From: Vojtech Trefny <vtrefny@redhat.com>
Date: Thu, 29 Jun 2023 08:36:48 +0200
Subject: vdo_stats: Remove unused libparted include

I have no idea where this came from, but we don't need (and never
did) libparted in vdo_stats.

Fixes: #916
(cherry picked from commit b870898b014f44a97fe996278210f59c57c1196b)
---
 src/plugins/vdo_stats.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/plugins/vdo_stats.c b/src/plugins/vdo_stats.c
index 7c8b8a4..620e972 100644
--- a/src/plugins/vdo_stats.c
+++ b/src/plugins/vdo_stats.c
@@ -18,7 +18,6 @@
  */
 
 #include <glib.h>
-#include <parted/parted.h>
 #include <blockdev/utils.h>
 
 #include "vdo_stats.h"
